packages/landing/src/config.map.ts

```diff
@@ -85,7 +85,7 @@ export class MapConfig extends Emitter<MapConfigEvents> {
     this.style = style ?? null;
     this.layerId = layerId.startsWith('im_') ? layerId.slice(3) : layerId;
     this.tileMatrix = tileMatrix;
-    this.debug = debug;
```

Comment:
can you also rename it to `isDebug` to show its a boolean

Reply:
Thanks fixed.

Reply:
was meaning `this.isDebug = debug` not the setter function name.

Reply:
Sorry, updated.